sat/cli/bootsys/parser.py

```diff
@@ -139,6 +139,18 @@ def _add_bos_template_options(subparser, action):
              'xname will be expanded into the node xnames for all nodes in that slot.'
     )
 
+    subparser.add_argument(
+        '--staged-session', action='store_true',
+        help=f'If specified, then create a "staged" BOS session. A staged session differs '
+             f'from a normal BOS session in that a staged session does not automatically '
+             f'change the state of any components targeted by the session templates. '
+             f'Instead, a staged session will update components\' "staged_state", which '
+             f'can later be applied through the command \'cray bos v2 applystage create\', '
+             f'or through an API call to BOS. When using this option, the command will '
+             f'not wait for completion of BOS sessions. This option cannot be used with '
+             f'BOS v1.'
```

Comment:
FYI, @jesseviola-hpe, we have to use BOS v2 if doing staged operations. I'm not sure if you were aware of that or not in your Argo workflow template implementation.

Reply:
Correct, I felt the prompt was pretty misleading in the staged case, since it might lead you to believe you were about to do an actual reboot/shutdown.